src/plots/power_vega.jl

```diff
@@ -143,19 +155,21 @@ end
 
 
 
-function plot_vega( case::Dict{String,<:Any};
```

Comment:
all the changes in plot_vega seem to be reverting the changes in the latest pull request to the dev branch.  Maybe try rebasing on dev?

Reply:
Rebased, should be fixed now.